index.js

```diff
@@ -70,8 +61,14 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: `Installing dependencies${opts.yarn ? ' using Yarn' : ''}`,
-				task: install(bin, opts.yarn)
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
We should implement a fallback as shown in the [example of listr](https://github.com/SamVerschueren/listr/blob/master/example.js#L38-L54). If `yarn` fails because the binary is not available, we should fall back to `npm` as well.